CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ release date when you use `npm version` (see `README.md`).
 
 ## [Unreleased]
 
+### Added
+
+- Simpler ways to use CLI:
+
+  - make the only useful `run` comand the default one
+  - default `location` argument to the current directory
```

Comment:
this seems more dangerous than helpful, to run current directory you just have to pass a dot "." that is not a super useful feature just to save writing a dot

Reply:
Dangerous in what sense?

This is comparable to `python -m http.server`, which uses current directory by default.

Reply:
ex. it can be pretty annoying if I run hoping it would show the help and then it even launches browser, and as said, what is the gain here? saving to type a dot?

Reply:
> it can be pretty annoying if I run hoping it would show the help

IDK, IMO `--help` is what you'd usually do.

> what is the gain here? saving to type a dot?

It can also be annoying if you launch it without arguments only to find out that you forgot the dot.

Reply:
But I see your point. It's a double-edged thing.

Reply:
you know what I would find annoying? that I run it without any arguments to see help, then it just run current directory I have to cancel, close browser tabs and type a `--help` now that is much more typing XD

Reply:
but anyways it is also ok to leave it like that